packages/design-system/src/components/Promo/Promo.js

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import getMergedClassNames from '../../utils/getMergedClassNames';
+
+import { Button } from '../Button';
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'promo';
+
+export const Promo = props => {
+  const {
+    buttonText,
+    children,
+    className,
+    header,
+    image,
+    light,
+    linkText,
+    onButtonClick,
+    onClose,
+    onLinkClick,
+    size,
+    img,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}--light`]: light,
+      [`${baseClass}--${size}`]: size,
+      [`${baseClass}`]: true,
+    }),
+    className
+  );
+  const shouldRenderLargeFooter = (buttonText || linkText)  && size === 'large'; 
+  const shouldRenderSmallOrMediumFooter = (buttonText || linkText)  && size !== 'large'; 
+
+  const footer = (
+    <div className={styles[`${baseClass}__footer`]}>
+      {buttonText && <Button primary onClick={onButtonClick} className={styles[`${baseClass}__button-text`]}>{buttonText}</Button>}
+      {linkText && <Button primary onClick={onLinkClick}>{linkText}</Button>}
+    </div>)
+
+  return (
+    <div className={mergedWrapperClassNames} {...restProps}>
+      <div className={styles[`${baseClass}__content`]}>
+        {img && <img src={img} className={styles[`${baseClass}__img`]} />}
+        <div className={styles[`${baseClass}__wrapper`]}>
+          <div className={styles[`${baseClass}__header`]}>{header}</div>
+          <div>{children}</div>
+            { shouldRenderSmallOrMediumFooter && footer }
+        </div>
+        { shouldRenderLargeFooter && footer }
+      </div>
+      <CloseIcon onClick={onClose} fill="#424d57" className={styles[`${baseClass}__close-icon`]}/>
```

Comment:
I forgot about that before but it would be awesome to make this focusable and choosable by a keyboard. Use button HTML tag as a parent for the icon